portfolio/src/main/webapp/script.js

```diff
@@ -71,6 +71,13 @@ function contact() {
 	console.log('Contact!'); 
 }
 
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(
```

Comment:
Nice. We're getting to the fun stuff. LMK when you deployed this PR. I wanna see how it looks.

Reply:
sounds good!